.github/workflows/workflow.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gatekeeper: [ "release-3.10", "release-3.11" ]
+        gatekeeper: [ "release-3.10", "release-3.11", "release-3.12" ]
```

Comment:
we should be okay to drop 3.10 since we support n-1 https://github.com/open-policy-agent/gatekeeper/blob/master/docs/Release_Management.md#supported-releases

Reply:
SG - Thanks for the link!